src/common/TopNav/components/NotiModal.jsx

```diff
@@ -0,0 +1,92 @@
+import { Typography } from '@mui/material';
+
+import styled from 'styled-components';
+
+const tempData = [
+  { title: '자유게시판', body: '제목입니다 글에 새로운 댓글이 달렸어요' },
+  { title: '자유게시판', body: '저 오늘 싸이봄ㅋㅋ 글이 글이asfasfasfas글이asfasfasfas 되었어요' },
+  { title: '자유게시판', body: '저 오늘 싸이봄ㅋㅋ 글이 인기글이 되었어요' },
+  { title: '자유게시판', body: '저 오늘 싸이봄ㅋㅋ 글이 인기글이 되었어요' },
+  { title: '자유게시판', body: '저 오늘 싸이봄ㅋㅋ 글이 인기글이 되었어요' },
+  { title: '자유게시판', body: '저 오늘 싸이봄ㅋㅋ 글이asfasfasfas 인기글이 되었어요' },
+  { title: '자유게시판', body: '저 오늘 싸이봄ㅋㅋ 글이 인기글이 되었어요' },
+];
+const NotiModal = () => {
+  return (
+    <Typography>
+      <ModalStyle>
+        <div className="main-title">알림</div>
+        <div className="divide"></div>
+
+        <div className="notis">
+          {tempData.map(data => {
+            return (
+              <>
+               <div className="noti">
+                  <img src="/assets/like.svg"/>
+                  <div className="body">
+                    <div className="title">{data.title}</div>
+                    <div className="text">{data.body}</div>
+                  </div>
+                </div>
+                <div className="divide"></div>
```

Comment:
```suggestion
const NotiLI = ({title, body}) => {
  return (
    <li className="noti">
      ...
    </li>
  )
}

...
tempData.map((data, index) => <NotiLi title={data.title} body={data.body} key={`noti-list-${index}`} />)
```

이렇게 컴포넌트 분리하는것이 좋을 것 같습니다 (기록용)